web/packages/teleport/src/DesktopSession/AlertDropdown.tsx

```diff
@@ -51,45 +51,43 @@ export function WarningDropdown({ warnings, onRemoveWarning }: Props) {
   return (
     <>
       <StyledButton
-        title={'Warnings'}
-        hasWarnings={warnings.length > 0}
+        title={'Alerts'}
+        hasAlerts={alerts.length > 0}
         px={2}
         onClick={toggleDropdown}
       >
         <Flex
           alignItems="center"
           justifyContent="space-between"
           color={
-            warnings.length
-              ? theme.colors.text.main
-              : theme.colors.text.disabled
+            alerts.length ? theme.colors.text.main : theme.colors.text.disabled
           }
         >
-          <Warning size={20} mr={2} /> {warnings.length}
+          <Warning size={20} mr={2} /> {alerts.length}
         </Flex>
       </StyledButton>
       {showDropdown && (
         <StyledCard
           mt={3}
           p={2}
           style={{
-            maxHeight: window.innerHeight / 4,
+            maxHeight: window.innerHeight / 3,
           }}
         >
           <Flex alignItems="center" justifyContent="space-between">
             <H3 px={3} style={{ overflow: 'visible' }}>
-              {warnings.length} {warnings.length > 1 ? 'Warnings' : 'Warning'}
+              {alerts.length} {alerts.length > 1 ? 'Alerts' : 'Alert'}
             </H3>
             <ButtonIcon size={1} ml={1} mr={2} onClick={toggleDropdown}>
               <Cross size="medium" />
             </ButtonIcon>
           </Flex>
-          <StyledOverflow flexWrap="wrap" gap={2}>
-            {warnings.map(warning => (
+          <StyledOverflow flexDirection="column" gap={2}>
```

Comment:
The flex property changes here are the biggest UI fix, the rest of the changes in this PR are mostly just renames.